programs/account-compression/src/instructions/insert_into_indexed_array.rs

```diff
@@ -0,0 +1,165 @@
+use std::collections::HashMap;
+
+use aligned_sized::aligned_sized;
+use anchor_lang::{prelude::*, solana_program::pubkey::Pubkey};
+use ark_ff::BigInteger256;
+use ark_serialize::CanonicalDeserialize;
+use light_hasher::Poseidon;
+use light_indexed_merkle_tree::array::IndexingArray;
+
+use crate::RegisteredProgram;
+
+#[derive(Accounts)]
+pub struct InsertIntoIndexedArrays<'info> {
+    /// CHECK: should only be accessed by a registered program/owner/delegate.
+    #[account(mut)]
+    pub authority: Signer<'info>,
+    pub registered_verifier_pda: Option<Account<'info, RegisteredProgram>>, // nullifiers are sent in remaining accounts. @ErrorCode::InvalidVerifier
```

Comment:
Since this is an optional account my idea is to check and throw the error in a different place at runtime.
see https://github.com/Lightprotocol/light-protocol/pull/457/files/2958c5034aa2dbd1f4f0845d8ec76773892fc5fe#diff-3131986c12409ae07fe68fb379bb72da1616097ca55938c837ffb130e83f3187R17

Reply:
Fair enough, then let's just remove the `@ErrorCode` from the comment.